builder/bundles.go

```diff
@@ -486,9 +500,11 @@ func installFilesystem(chrootDir string, packagerCmd []string, downloadRetries i
 			return err
 		}
 		if _, err = os.Stat(rpmFullPath); os.IsNotExist(err) {
+			log.Error(log.Mixer, "rpm not found for pkg: %s", pkgFull)
```

Comment:
These errors will eventually get caught in the topmost calling function by the fail() method. So having the log.Error in the fail() itself would suffice and we may not need it here.